cmd/flags.go

```diff
@@ -147,11 +150,19 @@ func CreateFlags(defaultPath string) []cli.Flag {
 		},
 		&cli.BoolFlag{
 			Name:  flgDNSDisableCP,
+			Usage: fmt.Sprintf("(deprecated) use %s instead.", flgDNSPropagationDisableANS),
+		},
+		&cli.BoolFlag{
+			Name:  flgDNSPropagationDisableANS,
 			Usage: "By setting this flag to true, disables the need to await propagation of the TXT record to all authoritative name servers.",
```

Comment:
```suggestion
			Usage: "By setting this flag, disables the need to await propagation of the TXT record to all authoritative name servers.",
```
Sorry if this is a nitpick, but that matches the style of the other descriptions better.

Reply:
I will do the opposite, just because I want to be really explicit on those flags, I'm afraid there is ambiguity.